packages/next-server/lib/router/router.ts

```diff
@@ -21,10 +21,10 @@ function toRoute(path: string): string {
   return path.replace(/\/$/, '') || '/'
 }
 
-export type BaseRouter = {
+export type BaseRouter<Query = any> = {
```

Comment:
should this be `Query = {}`. Anything intersected with `any` is `any`.

Reply:
Correct, missed it, will change it to `unknown`
![image](https://user-images.githubusercontent.com/9304194/60845688-c3702480-a1e5-11e9-8b3f-59291e3aa3c0.png)